lib/content-services/src/lib/search/components/search-filter-container/search-filter-container.component.scss

```diff
@@ -65,3 +65,7 @@
     min-width: 260px;
     padding: 0;
 }
+
+.margin-left-auto {
```

Comment:
ADF related classes should be prefixed with `adf`

Reply:
fixed